dap_prinz_green_jobs/pipeline/green_measures/industries/industry_measures_utils.py

```diff
@@ -0,0 +1,230 @@
+        "co",
+        "the",
+        "services",
+        "management",
+        "company",
+        "uk",
+        "c",
+        "llp",
+        "lp",
+        "international",
+        "group",
+        "cic",
+        "plc",
+    ]
+)
+
+
+def load_companies_house() -> pd.DataFrame():
+    """Downloads the Companies House dataset
+    :return: A dataframe of company information including name and SIC
+    :rtype: pd.DataFrame()
+    """
+    companies_house = pd.read_csv(
+        "s3://prinz-green-jobs/inputs/data/industry_data/BasicCompanyDataAsOneFile-2023-05-01_key_columns_only.csv"
+    )
+    return companies_house
+
+
+def load_industry_ghg() -> pd.DataFrame():
+    """Downloads a dataset of greenhouse gas emissions per SIC
+    :return: A dataframe of SIC and greenhouse gas emissions
+    :rtype: pd.DataFrame()
+
+    TO DO: clean this dataset - there are SIC codes in there like '20.12+20.2'
+    """
+    emissions_data = pd.read_excel(
+        "s3://prinz-green-jobs/inputs/data/industry_data/atmosphericemissionsghg.xlsx",
+        sheet_name="GHG total",
+        skiprows=3,
+    )
+    emissions_data.reset_index(inplace=True)
+    emissions_data = emissions_data.loc[list(range(0, 21)) + list(range(26, 156))]
+
+    emissions_data["Unnamed: 0"] = emissions_data["Unnamed: 0"].apply(
+        lambda x: x if isinstance(x, str) else "0" + str(x) if x < 10 else str(x)
+    )
+
+    return emissions_data
+
+
+def load_sic() -> pd.DataFrame():
+    """Downloads a dataset of greenhouse gas emissions per SIC
+    :return: A dataframe of SIC and greenhouse gas emissions
+    :rtype: pd.DataFrame()
+    """
+    sic_data = pd.read_excel(
+        "s3://prinz-green-jobs/inputs/data/industry_data/publisheduksicsummaryofstructureworksheet.xlsx",
+        sheet_name="reworked structure",
+    )
+
+    return sic_data
+
+
+def clean_company_name(
+    name: str,
+    word_mapper: dict = {},
+    company_stop_words: set = company_stop_words,
+) -> str:
+    """Clean the company name so it can be matched across datasets
+
+    There is lots of different ways to write company names, so this normalises
+    across datasets for matching. e.g. "Apple ltd." and "apple limited"
+
+    :param name: A company name
+    :type: str
+    :param word_mapper: A dict of words to replace with others (e.g. {"ltd": "limited"})
+    :type: dict
+    :param company_stop_words: words to be removed from the name
+    :type: set
+    :return: A cleaned company name
+    :rtype: str
+    """
+
+    name = str(name)
+    name = re.sub(r"[^\w\s]", "", name)
+    name = " ".join(name.split())  # sort out double spaces and trailing spaces
+    name = name.lower()
+
+    name_words = name.split()
+    words = [
+        word_mapper.get(word, word)
+        for word in name_words
+        if word not in company_stop_words
+    ]
+
+    name = " ".join(words)
+
+    return name
+
+
+def get_green_industry_measure(company_name):
+    return random.choice(["Green", "Not green"])
```

Comment:
This is what I'm using in `18_greenmeasures` - but looks like it should be what you're doing in extract_industry_measures for a given company_name instead a list of company_names - happy to do that it that pull request

Reply:
ah yes thats right! eyh - maybe it makes sense to incorporate some of whats in `extract_industry_measures` into the GreenMeasures class instead then